lib/extensions/descendant_loader.rb

```diff
@@ -268,6 +268,7 @@ def clear
 
 ActiveRecord::Base.singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
 ActiveSupport::Dependencies.send(:prepend, DescendantLoader::AsDependenciesClearWithLoader)
+ActsAsArModel.singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
I know why but this makes me sad.  😢 

How about we wrap this private-ness in a register method, that way we can register classes with the descendant_loader?  Good idea?  Bad idea?